radio/src/datastructs_private.h

```diff
@@ -693,11 +693,12 @@ PACK(struct ModelData {
   uint8_t   disableTelemetryWarning:1;
   uint8_t   showInstanceIds:1;
   uint8_t   checklistInteractive:1;
+  uint8_t   crsfArmingMode:1;
```

Comment:
Why is this in the Model data and not the Module data?

Reply:
Consider the rather rare corner case where an internal and external module are active at the same time e.g. to build a 900Mhz/2.4GHz redundancy. Kinda nonsense with GemX being around but we currently don't reject a dual module setup even if telemetry is a problem.

What we do currently for this corner case wrt arming is to have both modules listening to CH5. The same CH5 data is sent to the internal and external module. Both modules will arm/disarm synchronously. So arming is already effectively a model property rather than a module property.

We could make it module specific or which I'd prefer (gut feeling) leave it model specific meaning in the case of two active modules both would be armed/disarmed synchronously in both arming modes